### PR TITLE
Project for UI documentation

### PR DESCRIPTION
**Pull request by le1nux**

### CODE REVIEW

File: ML_BOARD.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+## MLBoard
+
+In the GIF images below, we present a detailed walkthrough of an example run using the above provided tutorial of `grid_search_example`. Follow the sequential execution commands to seamlessly initiate and monitor the experiment.
```
Comment (le1nux) [inline ML_BOARD.md:3]: would be great  link the example

File: ML_BOARD.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+3. **Pipeline Execution:** Run the Python file to execute the pipeline.
+4. **Frontend Visualization:** Launch the frontend (mlboard) to visualize metrics and loss for different experiments.
+
+<div align="center">
+<img src="mlboard_gifs/ML_Board_Start_Things.gif" width="100%" />
+</div>
+
+### Monitoring and Analysis
+
+After executing the example run, proceed to monitor and analyze the experiment using the `ml_board`. First execute the ml_board command with the example run ID, and copy the provided URL. Paste the URL in your browser to access the frontend.
```
Comment (le1nux) [inline ML_BOARD.md:16]: would be great to also add the commands here so that they can be copied

**Review by le1nux — COMMENTED** [review]
Great tutorial for MLboard. 👍  After addressing my two comments we can merge it :) 

Could you also please reference the tutorial in the README.md?

**Comment by vijulshah** [comment]
Hey @le1nux , I have pushed the new changes for the documentation. Please have a look and let me know if it needs any changes.

**Review by le1nux — COMMENTED** [review]
LGTM :) Nice plots. Merging

**Pull request merged**